### PR TITLE
Added my lessons learned from executing the instructions.

### PR DESCRIPTION
**Pull request by pheye1618**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,21 +14,25 @@ If you find any problems with the code or have any ideas on improving it, please
...
--   [Getting Started](#getting-started)
--   [High-level Architecture](#high-level-architecture)
--   [Preparing Your Environment](#preparing-your-environment)
--   [Running MLflow](#running-mlflow)
--   [Training The Model](#training-the-model)
--   [Visualizing Pipeline Results](#visualizing-pipeline-results)
--   [Deploying The Model](#deploying-the-model)
--   [Monitoring The Model](#monitoring-the-model)
--   [Production Pipelines in Amazon Web Services](#production-pipelines-in-amazon-web-services)
+- [Building Machine Learning Systems](#building-machine-learning-systems)
```
Comment (svpino) [inline README.md:17]: Let's keep "Building Machine Learning Systems" and "Table of Contents" outside of the Table of Contents.

File: README.md [hunk 2]
```diff
@@ -14,21 +14,25 @@ If you find any problems with the code or have any ideas on improving it, please
...
+  - [Visualizing Pipeline Results](#visualizing-pipeline-results)
+  - [Deploying The Model](#deploying-the-model)
+  - [Monitoring The Model](#monitoring-the-model)
+  - [Production Pipelines in Amazon Web Services](#production-pipelines-in-amazon-web-services)
     - [Running a remote MLflow server in EC2](#running-a-remote-mlflow-server-in-ec2)
     - [Deploying the model to SageMaker](#deploying-the-model-to-sagemaker)
     - [Monitoring the model in SageMaker](#monitoring-the-model-in-sagemaker)
     - [Deploying to AWS Managed Services](#deploying-to-aws-managed-services)
+      - [Running the Training pipeline remotely](#running-the-training-pipeline-remotely)
+      - [Running the Deployment pipeline remotely](#running-the-deployment-pipeline-remotely)
```
Comment (svpino) [inline README.md:33]: Let's remove these 3rd level items.

File: README.md [hunk 3]
```diff
@@ -114,7 +118,7 @@ After the server is running, modify the `.env` file inside the repository's root
 export $((echo "MLFLOW_TRACKING_URI=http://127.0.0.1:5000" >> .env; cat .env) | xargs)
 ```
 
-## Training The Model
+## Training The Model {#training-the-model}
```
Comment (svpino) [inline README.md:121]: You shouldn't need the {#training-the-model} portion of the TOC to work correctly in GitHub or Visual Studio Code.

File: README.md (hunk 1 above)
Comment (pheye1618) [reply README.md:17]: Sure.

File: README.md (hunk 2 above)
Comment (pheye1618) [reply README.md:33]: Sure.

File: README.md (hunk 3 above)
Comment (pheye1618) [reply README.md:121]: Agreed.

**Pull request closed without merging**